superset/security.py

```diff
@@ -111,6 +111,10 @@ def all_datasource_access(self):
     def all_database_access(self):
         return self.can_access('all_database_access', 'all_database_access')
 
+    def all_database_access(self):
```

Comment:
Isn't this the very same method as the one above?

Reply:
Yes it is. Went ahead and removed it. Must have been duplicated when I was merging in new changes from master into my branch.